src/index.ts

```diff
@@ -56,7 +56,7 @@ export function convertTsConfig(
       } satisfies swcType.ModuleConfig,
       jsc: {
         externalHelpers: importHelpers,
-        target: target as swcType.JscTarget,
+        target: targetType(target as any),
```

Comment:
Aside, but note that the use of `tsType.CompilerOptions` within this repo is incorrect. `CompilerOptions.target` is a ts enum with numeric values, while the value loaded from the tsconfig.json is a string. This is why the extra `target as any` is required.